taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import useFetch from "../../components/useFetch";
+import { API_URL } from "../../constants";
+import ListAllEntryProperties from "./ListAllEntryProperties";
+import ListAllOtherProperties from "./ListAllOtherProperties";
+
+// Used for rendering node information
+// If node is an "entry": Relations, translations, comments and properties are rendered
+// If node is an "stopword/synonym": Stopwords/synonyms, language and comments are rendered
+// If node is "header/footer": Comments are rendered
+
+const AccumulateAllComponents = ({ id }) => {
+    
+    // Finding URL to send requests
+    let url = API_URL;
+    let isEntry = false;
+    if (id.startsWith('__header__')) { url += 'header/' }
```

Comment:
please somewhere add a comment like `// the id looks like: '....'` so that it is easier to understand what is in there